src/vsphere_cpi/Rakefile

```diff
@@ -0,0 +1,69 @@
+require 'time'
+require 'shellwords'
```

Comment:
I don't think we need either `time` or `shellwords` in here.

Reply:
Removed. Fixed